Src/PerformanceCollector/Shared/QuickPulseTelemetryModule.cs

```diff
@@ -318,7 +318,7 @@ private void InitializeServiceClient(TelemetryConfiguration configuration)
         private static string GetInstanceName(TelemetryConfiguration configuration)
         {
             // we need to initialize an item to get instance information
-            var fakeItem = new MetricTelemetry();
```

Comment:
It's ok. You can even create a new implementation of ITelemetry so it will not be sent to the server even if somehow will reach the channel.